standard-hackathon-rules.md

```diff
@@ -10,14 +10,14 @@ Remember that hackathons are like marathons. Some people go to compete but most
 
 ## The rules of the competition
 
-1. There is no maximum or minimum team size. As a participant, you should make sure to check how many prizes are available per team. There is usually a limited number of prizes for each challenge. So if you form a large team and win a challenge, there might not be enough prizes for everyone on your team.
+1. There is no maximum or minimum team size, however we do recommending having up to 4 team members to maximize the hackathon experience. As a participant, you should make sure to check how many prizes are available per team. There is usually a limited number of prizes for each challenge. So if you form a large team and win a challenge, there might not be enough prizes for everyone on your team.
 2. Teams should be made up exclusively of students (or recent graduates within one year of having graduated) who are not organizers, volunteers, judges, sponsors, or in any other privileged position at the event. 
 3. All team members should be present at the event. Leaving the venue for some time to hack elsewhere is fine.
 4. Teams can of course gain advice and support from organizers, volunteers, sponsors, and others.
-5. All work on a project should be done at the hackathon.
-6. Teams can use an idea they had before the event.
+5. All work on a project that will be demo'd should only have been created at the current hackathon.
+6. Teams can use an idea they had before the event.  Code built prior to the event can be used but must be properly documented.  Only new code will be considered eligible at the current hackathon.  
 7. Teams can work on ideas that have already been done. Hacks do not have to be “innovative”. If somebody wants to work on a common idea they should be allowed to do so and should be judged on the quality of their hack. These days it’s hard to find something that’s fully original and teams might not know an idea has been done before anyway.
-8. Teams can work on an idea that they have worked on before (as long as they do not re-use code or other project materials).
+8. Teams can work on an idea that they have worked on before (as long as they do not re-use code or other project materials without proper documentation).
```

Comment:
Kinda the same point at #6. Maybe #6 can be on the _idea_ itself versus #8 can be focused on the code itself

Reply:
No documentation thing

Reply:
++ if worked on this idea before, you must disclose that you've created something similar to it before to your judges and submission.